model/qti/ImportService.php

```diff
@@ -328,7 +328,7 @@ public function importQTIPACKFile($file, core_kernel_classes_Class $itemClass, $
                         }
                         
                         // replace uri if changed
-                        if ($auxPath != $info['uri']) {
+                        if ($auxPath != ltrim($info['uri'], '/')) {
```

Comment:
Yes, i know.
This slash added here:
https://github.com/oat-sa/extension-tao-item/blob/develop/models/classes/media/LocalItemSource.php#L162